liblangutil/ErrorReporter.cpp

```diff
@@ -28,6 +28,8 @@ using namespace std;
 using namespace solidity;
 using namespace solidity::langutil;
 
+ErrorCase solidity::langutil::operator"" _error(unsigned long long _error) { return ErrorCase{ _error }; }
```

Comment:
Is it a case or an identifier?

Reply:
> Is it a case or an identifier?

It is an **error case**, as for now we are tagging places where errors are reported, not errors themselves. The difference is small and appears rather in corner cases (see e.g. the last lambda in **TypeChecker.cpp**).

When/if we refactor all such corner cases, we will get unique **error identifiers**. A good thing is that `ErrorCase` is not used outside of `ErrorReporter`, so we can rename it easily.

Reply:
I think identifier is still easier to understand. Case is a rather abstract concept while identifier is well understood. I would say if we plan to rename it anyway, we should do it now - it is a work in progress and nothing is perfect.